src/components/TradeWidget/TokenRow.tsx

```diff
@@ -236,13 +254,31 @@ const TokenRow: React.FC<Props> = ({
     <Wrapper>
       <div>
         <strong>{selectLabel}</strong>
-        <span>
+        <div>
           {!readOnly && (
-            // TODO: Implement deposit in Trade widget. When ready, show also the Wrap Ether button
-            //  https://github.com/gnosis/dex-react/issues/610
-            <Link className="btn" to="/wallet">
-              + Deposit
-            </Link>
+            <>
+              <div className="btn" onClick={(): void => showForm('deposit')}>
+                + Deposit
+              </div>
```

Comment:
In `/wallet` page we allow depositing only when [token is enabled](https://github.com/gnosis/dex-react/blob/b477bb795a5864b10e0020b1b797df94746cc67b/src/components/DepositWidget/Row.tsx#L150).  Should do same here.

Plus can't deposit a not enabled token (without allowance)

Reply:
Unresolving

Reply:
ok now @alfetopito @Velenir

Reply:
Niiiice!

Reply:
Niiiiiiiiiiiice!